files/arc/ppark.json.txt

```diff
@@ -0,0 +1,23 @@
+#include "global.h"
+
+struct PalParkMonsData {
+    u8 land_sector;
+    u8 water_sector;
+    u8 score;
+    u8 encounter_rate;
+    u8 unk4;
+    u8 unk5;
+};
+
+const struct PalParkMonsData gData[] = {
+{% for mon in mons %}
+    { // {{mon.species}}
```

Comment:
remove comment?

Reply:
It's there in case some masochist decides they want to inspect the intermediate C file.